kratos/solving_strategies/strategies/explicit_solving_strategy.h

```diff
@@ -0,0 +1,681 @@
+     * @details It is designed to be called ONCE to verify that the input is correct.
+     */
+    virtual int Check()
+    {
+        KRATOS_TRY
+
+        // Check if displacement var is needed
+        if (mMoveMeshFlag == true) {
+            VariableUtils().CheckVariableExists<>(DISPLACEMENT, GetModelPart().Nodes());
+        }
+
+        // Elements check
+        for (const auto& r_elem :  GetModelPart().Elements()) {
+            const auto& r_process_info = GetModelPart().GetProcessInfo();
+            r_elem.Check(r_process_info);
+        }
+
+        // Conditions check
+        for (const auto& r_cond :  GetModelPart().Conditions()) {
+            const auto& r_process_info = GetModelPart().GetProcessInfo();
+            r_cond.Check(r_process_info);
+        }
+
+        return 0;
+
+        KRATOS_CATCH("")
+    }
+
+    ///@}
+    ///@name Input and output
+    ///@{
+
+    /// Turn back information as a string.
+    virtual std::string Info() const
+    {
+        return "ExplicitSolvingStrategy";
+    }
+
+    /// Print information about this object.
+    virtual void PrintInfo(std::ostream &rOStream) const
+    {
+        rOStream << Info();
+    }
+
+    /// Print object's data.
+    virtual void PrintData(std::ostream &rOStream) const
+    {
+        rOStream << Info();
+    }
+
+    ///@}
+
+protected:
+    ///@name Protected static Member Variables
+    ///@{
+
+    // Level of echo for the solving strategy
+    int mEchoLevel;
+
+    // Settings for the rebuilding of the DOF set
+    int mRebuildLevel;
+
+    ///@}
+    ///@name Protected member Variables
+    ///@{
+
+    ///@}
+    ///@name Protected Operators
+    ///@{
+
+    ///@}
+    ///@name Protected Operations
+    ///@{
+
+    /**
+     * @brief Calculate the explicit update
+     * This method is intended to implement the explicit update calculation
+     * Note that it has to be implemented according to the explicit scheme in a derived class
+     */
+    virtual void SolveWithLumpedMassMatrix()
+    {
+        KRATOS_ERROR << "Calling the base ExplicitSolvingStrategy SolveWithLumpedMassMatrix(). Implement the specific explicit scheme solution update in a derived class" << std::endl;
+    }
+
+    /**
+     * @brief Get the Delta Time object
+     * This method returns the DELTA_TIME from the ProcessInfo container
+     * @return const double
+     */
+    virtual inline double GetDeltaTime()
+    {
+        return GetModelPart().GetProcessInfo().GetValue(DELTA_TIME);
+    }
+
+    ///@}
+    ///@name Protected  Access
+    ///@{
+
+    ///@}
+    ///@name Protected Inquiry
+    ///@{
+
+    ///@}
+    ///@name Protected LifeCycle
+    ///@{
+
+private:
+    ///@}
+    ///@name Static Member Variables
+    ///@{
+
+    ///@}
+    ///@name Member Variables
+    ///@{
+
+    ModelPart &mrModelPart;
+
+    bool mMoveMeshFlag;
+
+    bool mInitializeWasPerformed = false;
+
+    bool mInitializeSolutionStepWasPerformed = false;
```

Comment:
```suggestion
```

can we get rid of those please?

I think they are really bad practice and a leftover of the `Solver` function